apps/src/templates/teacherHomepage/ManageSections.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import CollapsibleSection from './CollapsibleSection';
+// import SectionsTable from './SectionsTable';
+import SetUpMessage from './SetUpMessage';
+import i18n from "@cdo/locale";
+
+const ManageSections = React.createClass({
+  propTypes: {
+    sections: React.PropTypes.array,
+  },
+
+  render() {
+    const { sections } = this.props;
+
+    return (
+      <CollapsibleSection
+        header={i18n.mySections()}
+        linkText={i18n.editSections()}
+        link="https://code.org/teacher-dashboard#/sections"
+      >
+      {sections ? (
+        <div>
+          This is where the sections table will go.
+        </div>
+      ) : (
+        <SetUpMessage type="sections"/>
+      )}
```

Comment:
nit: I always find this syntax difficult to follow in JSX. I prefer an approach like this:
```jsx
{sections && (
  <div>Contents of my div</div>
)}
{!sections && (
  <SetUpMessage type="sections"/>
)}